roles/openshift_hosted/README.md

```diff
@@ -33,13 +33,20 @@ variables also control configuration behavior:
 
 | Name                                         | Default value | Description                                                                  |
 |----------------------------------------------|---------------|------------------------------------------------------------------------------|
-| openshift_hosted_registry_glusterfs_swap     | False         | Whether to swap an existing registry's storage volume for a GlusterFS volume |
-| openshift_hosted_registry_glusterfs_swapcopy | True          | If swapping, also copy the current contents of the registry volume           |
+| openshift_hosted_registry_storage_glusterfs_swap     | False         | Whether to swap an existing registry's storage volume for a GlusterFS volume |
+| openshift_hosted_registry_storage_glusterfs_swapcopy | True          | If swapping, also copy the current contents of the registry volume           |
+| openshift_hosted_registry_storage_glusterfs_ips      | []            | If we use external gluster for Registry backend we need to provide list of IP's for EP creation|
+| openshift_hosted_registry_storage_glusterfs_path     | []            | If we use external gluster we need to provide volume name from gluster cluster for mounting|
```

Comment:
Do not add this, this is covered in the GlusterFS role.

Reply:
If user do not deploy gluster and want just to use it, why he/she should go and read readme of glsuterFS role? :) My vote is to keep it here too.